assets/html/scss/documenter-light.scss

```diff
@@ -40,4 +40,57 @@ code.language-julia-repl > span.hljs-meta {
 
 // Workaround to compile in highlightjs theme, so that we could have different
 // themes for both
-@import "highlightjs/default"
+@import "highlightjs/default";
+
+.search-result-link {
+  border-radius: 0.7em;
+  transition: all 300ms;
+}
+
+.search-result-link:hover, .search-result-link:focus {
+  background-color: rgba(0, 128, 128, 0.1);
+}
+
+.search-result-link .property-search-result-badge {
+  transition: all 300ms;
+}
+
+.search-result-link:hover .property-search-result-badge, .search-result-link:focus .property-search-result-badge {
+  color: #f1f5f9 !important;
+  background-color: #333 !important;
+}
+
+.property-search-result-badge {
+  padding: 0.15em 0.5em;
+  font-size: 0.8em;
+  font-style: italic;
+  text-transform: none !important;
+  line-height: 1.5;
+  color: whitesmoke;
+  background-color: #33415580;
+  border-radius: 0.6rem;
+}
+
+.search-result-highlight {
+  background-color: hsl(48, 100%, 67%);
+}
+
+.search-divider {
```

Comment:
Hmm, this won't on `a` tag because we are rounding off the borders so it will look like this.

![image](https://github.com/JuliaDocs/Documenter.jl/assets/80420115/4a3059ea-69c6-4de6-82a7-4eddcfdbab2a)

Two ways to solve this,

1. Remove the `border-radius` from `a` tag
2. Keep this as it is.

Reply:
Fair point. Let's keep it as is.